iModelCore/GeomLibs/geom/src/structs/cpp/refmethods/refdellipse3d.cpp

```diff
@@ -1749,23 +1750,14 @@ double          sweepIn
 
 
 /*-----------------------------------------------------------------*//**
-@description Initialize an elliptical arc from 3 points.
-
- @param [out] pEllipse initialized ellipse
- @param [in] start start point
- @param [in] middle mid point
- @param [in] end end point
- @return true if the three points are valid, false if colinear.
  @group "DEllipse3d Initialization"
```

Comment:
The `@group` can also be moved to the header doc.

Reply:
Are you sure? All `@group` for other API is only in the cpp file

Reply:
Yes. The `@group` tag is used (or at least it has been used historically) by various documentation processing scripts to link related methods in external documentation pages/files. As these scripts operate only on the header files, this tag is not needed in .cpp source files. The headers for public consumption are in GeomLibs\PublicAPI and they are chock full of these tags.

Reply:
ok so when you say move to the header doc you dont mean dellipse3d.h file. so which file do you mean?

Reply:
Ah, ok, I see now. You can just remove it from the source file. It is apparently not used anymore, as all those occurrences in PublicAPI are for methods in "capi" subfolders, for legacy C API functions that aren't necessarily exposed in modern C++ classes.

Reply:
But all other functions in refdellipse3d.cpp have `@group` 
Does not make sense to keep these 2 as well?

Reply:
AFAIK, `@group` is an historical artifact. When I'm already modifying a method in .cpp, I move its doc to its .h file and leave behind a minimal comment block to set it visually apart. Everything else in the .cpp doc block is either redundant or extraneous.